src/dodal/devices/aperturescatterguard.py

```diff
@@ -171,14 +192,13 @@ async def _get_current_aperture_position(self) -> SingleAperturePosition:
         assert isinstance(self.aperture_positions, AperturePositions)
         current_ap_y = await self.aperture.y.user_readback.get_value(cached=False)
         robot_load_ap_y = self.aperture_positions.ROBOT_LOAD.location.aperture_y
-        tolerance = self.TOLERANCE_STEPS * await self.aperture.y.deadband.get_value()
         if await self.aperture.large.get_value(cached=False) == 1:
             return self.aperture_positions.LARGE
         elif await self.aperture.medium.get_value(cached=False) == 1:
             return self.aperture_positions.MEDIUM
         elif await self.aperture.small.get_value(cached=False) == 1:
             return self.aperture_positions.SMALL
-        elif current_ap_y <= robot_load_ap_y + tolerance:
+        elif current_ap_y <= robot_load_ap_y + self.aperture_positions.tolerances.ap_y:
```

Comment:
Is this definitely the correct variable? The issue says we should be using the scatterguard tolerance (`sg_y_tolerance`), but this is using the `miniap_y_tolerance`

Reply:
where do you see this in the issue? we're comparing positions for the aperture device so I don't see why we wouldn't use the tolerances for it too